File: crop-screenshots.py
from PIL import Image

# Crop Culture Alberta Instagram screenshot
img1 = Image.open(r'c:\Users\Admin\Documents\culture-media-agency\public\instagram-culturealberta.png')
# Remove top 60 pixels (status bar)
cropped1 = img1.crop((0, 60, img1.width, img1.height))
cropped1.save(r'c:\Users\Admin\Documents\culture-media-agency\public\instagram-culturealberta.png')
print("Culture Alberta screenshot cropped")

# Crop Culture YYC Instagram screenshot
img2 = Image.open(r'c:\Users\Admin\Documents\culture-media-agency\public\instagram-cultureyyc.png')
# Remove top 60 pixels (status bar)
cropped2 = img2.crop((0, 60, img2.width, img2.height))
cropped2.save(r'c:\Users\Admin\Documents\culture-media-agency\public\instagram-cultureyyc.png')
print("Culture YYC screenshot cropped")

File: crop-screenshots.js
const sharp = require('sharp');
const path = require('path');

async function cropImages() {
    try {
        // Crop Culture Alberta screenshot
        await sharp('public/instagram-culturealberta.png')
            .extract({ left: 0, top: 60, width: 400, height: 800 - 60 })
            .toFile('public/instagram-culturealberta-cropped.png');

        // Replace original with cropped
        const fs = require('fs');
        fs.renameSync('public/instagram-culturealberta-cropped.png', 'public/instagram-culturealberta.png');

        console.log('Culture Alberta screenshot cropped');

        // Crop Culture YYC screenshot
        await sharp('public/instagram-cultureyyc.png')
            .extract({ left: 0, top: 60, width: 400, height: 800 - 60 })
            .toFile('public/instagram-cultureyyc-cropped.png');

        fs.renameSync('public/instagram-cultureyyc-cropped.png', 'public/instagram-cultureyyc.png');

        console.log('Culture YYC screenshot cropped');
        console.log('Done! Status bars removed.');
    } catch (error) {
        console.error('Error:', error);
    }
}

cropImages();
